activerecord/test/cases/database_configurations/hash_config_test.rb

```diff
@@ -69,6 +69,21 @@ def test_idle_timeout_nil_when_less_than_or_equal_to_zero
         config = HashConfig.new("default_env", "primary", idle_timeout: "0")
         assert_nil config.idle_timeout
       end
+
+      def test_default_schema_dump_value
+        config = HashConfig.new("default_env", "primary", schema_dump: nil)
```

Comment:
Can you change these tests to use an explicit hash? This method doesn't accept kwargs.

```suggestion
        config = HashConfig.new("default_env", "primary", { schema_dump: nil })
```